rails/db/schema.rb

```diff
@@ -1922,15 +1922,6 @@
 
   add_index "portal_teachers", ["user_id"], :name => "index_portal_teachers_on_user_id"
 
-  create_table "recent_collections_pages", :force => true do |t|
-    t.integer  "recent_project_id", :null => false
-    t.integer  "teacher_id",        :null => false
-    t.datetime "created_at",        :null => false
-    t.datetime "updated_at",        :null => false
-  end
-
-  add_index "recent_collections_pages", ["teacher_id"], :name => "index_recent_collections_pages_on_teacher_id"
-
```

Comment:
I added this back in recently. It looks like you don't have this in your database, but I do. 
Can you look into it to see which one of our databases is correct?